app/main.ipynb

```diff
@@ -56,33 +59,57 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
-   "outputs": [],
    "source": [
-    "# write your code here (for Python default syntax)"
+    "arr = np.arange(1000)\n",
+    "sys.getsizeof(arr)"
```

Comment:
The array `arr` is created using `np.arange(1000)`, which does not meet the requirement of containing zeros. Consider using `np.zeros(1000)` to create an array of zeros.